### DISCUSSION TITLE
월드 개발 중 로그인을 할 수가 없습니다.

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#1212] [5. Q&A]
Publish 버튼 오른쪽에 있는 역삼각형 아이콘을 눌렀을 때 로그인하는 버튼만 사라지는 현상으로 인해서 개발이 제한되고 있습니다. 

유니티 개발 환경은 2020.3.9f1입니다.

유니티 재설치와 제페토 템플릿 재설치 등을 진행했는데도 문제가 전혀 해결되지 않고 있습니다.

혹시 이러한 문제를 겪고 해결하신 분들 있나요? 도움이 필요합니다.

**Comment by YSHFM** [comment]
조금 다른 현상으로 저는 다른 ui가 몇개 안보이는 문제가 있어서 패키지매니저에서 
![image](https://user-images.githubusercontent.com/106574541/186118704-bde2a867-c8ec-4b89-bd91-497bbb8e9ba7.png)
해당 메뉴로 체크한 뒤 저는 미설치 된 패키지가 있어서 해당 패키지 Install 후 해결되었습니다.

**Reply by zlmrzlo** [reply]
저의 경우에는 유니티가 비정상적으로 종료되면서 레지스트리에 문제가 생겼던 것이었습니다. 레지스트리를 모두 삭제하고 다시 부팅하니 원래대로 돌아왔습니다. 답변 감사합니다!
